src/library/crate/cratestorage.cpp

```diff
@@ -441,6 +457,26 @@ CrateTrackSelectResult CrateStorage::selectTrackCratesSorted(TrackId trackId) co
 }
 
 
+CrateTrackSelectResult CrateStorage::selectTracksSortedByCrateNameLike(const QString& crateName) const {
+    FieldEscaper escaper(m_database);
```

Comment:
As already mentioned, please bind crateName as a named parameter to the query using bindValue(). FieldEscaper then becomes obsolete here.